client/analysisui/iconsupport.css

```diff
@@ -0,0 +1,40 @@
+/*textbox sizing*/
+.jmv-option-icon[data-icon=ContinuousVariable] {
```

Comment:
although i guess this wants to map to the assets directory, and so something with the group might be better. perhaps like the urls:

```
.jmv-option-icon[data-icon=variable-continuous] {
    ...
}
```